notebooks/pandas_04_time_series_data.md

```diff
@@ -356,28 +356,34 @@ These strings can also be combined with numbers, eg `'10D'`...
 </div>
 
 ```{code-cell} ipython3
-data.resample('M').mean().plot() # 10D
+data.resample('ME').mean().plot() # 10D, MS
 ```
 
 <div class="alert alert-success">
 
 **EXERCISE**
 
-Plot the monthly standard deviation of the columns
+Plot the monthly standard deviation of the columns. Use the last day of each month to represent the month.
+
+<details><summary>Hints</summary>
+
+* Pandas has both a [`ME` and `MS` month (offset)](https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#dateoffset-objects) to represent the month by the 'calendar month end' (`ME`) or 'calendar month start' (`MS`) respectively.
```

Comment:
( I am not a great fan of this complication .. instead of just having a default "M" mean month end, but we'll have to live with this)

Wondering if this is worth pulling out of a hidden hint

Reply:
-> I agree, the intuitive option is no longer available...

I propose to keep it in the hints, as the current online documentation on these dateoffsets explains the ME/MS situation and participants are not always aware about the previous situation (so it is a hint like any other hint on the syntax)